nss-day-cohort-41/react-nutshell-thepecans
# Ba login

# Description
Login feature works.  Currently no registration section, but you can login as a user from the database.  Logging-in will set "credentials" in session storage to the userId from the database.
## Type of change
Please delete options that are not relevant.

# Testing Instructions
navigate to "http://localhost:3000/login"  in the browser. Open session storage in console.  Type in requested login information, ensuring that the username and password you provide are identical with that of a user from the database. verify that the number in session storage is identical to the user's id.
# Checklist:
